packages/web-components/fast-foundation/src/button/button.template.ts

```diff
@@ -10,6 +10,7 @@ export function buttonTemplate<T extends FASTButton>(
     options: ButtonOptions = {}
 ): ElementViewTemplate<T> {
     return html<T>`
+    <template role="button" aria-describedby="${x => x.ariaDescribedby}">
```

Comment:
This will have much more wide ranging effects when fixing this. If the proposal is to do this, which is certainly one thing which could be done, I would suggest a wholesale move to remove the internal button.

Reply:
I thought about doing that but found it too extreme.
having a `button` inside a `role="button"` is not great though, I agree. what other "wide ranging effects" are you thinking of?